daprdocs/content/en/reference/components-reference/supported-cryptography/json-web-key-sets.md

```diff
@@ -0,0 +1,40 @@
+---
+type: docs
+title: "JSON Web Key Sets (JWKS)"
```

Comment:
```suggestion
title: "JSON Web Key Sets (JWKS) (Built-in)"
```

Reply:
After I push these changes, let me know if you think we still need to add to the title - I don't think we do